nimbus/src/nimbus.idl

```diff
@@ -87,12 +87,31 @@ interface NimbusClient {
     [Throws=Error]
     sequence<EnrollmentChangeEvent> set_global_user_participation(boolean opt_in);
 
-    // Updates the list of experiments from the server. After calling this, the
-    // list of active experiments might change (there might be new experiments,
-    // or old experiments might have expired)
+    // Updates the list of experiments from the server.
+    // This method is deprecated, in favour of calling `fetch_experiments()` and then
```

Comment:
I guess more a question for @travis79, but do we really need to deprecate (ie, can we just also make the somewhat trivial PR for a-c?) If we do, it's probably worth a `#[deprecated = "blah"]` decoration on the implementation?

Reply:
Yeah, with the lack of consumers I think we can be safe to move fast and skip the deprecation.

Reply:
This will cause unavoidable build breakage for android-components.